py/requirements.txt

```diff
@@ -2,8 +2,8 @@ numpy
 packaging
 pybind11==2.6.2
 --extra-index-url https://download.pytorch.org/whl/nightly/cu121
-torch==2.1.0.dev20230703+cu121
-torchvision==0.16.0.dev20230703+cu121
+torch>=2.1.0.dev,<2.2.0
+torchvision==0.16.0.dev,<0.17.0
```

Comment:
Should be updated to `torchvision>=0.16.0.dev,<0.17.0`, to allow any `0.16.0.dev` variant.